app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -385,4 +396,12 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
             cutout = true
         return cutout
     }
+
+    private fun waitForConnection() {
+        Handler().postDelayed({
+            if (!isConnected) {
+                showError()
```

Comment:
Can we let this screen automatically hide if `connection-status` event is fired that we're `connected` ?

Reply:
Done.